src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -65,6 +65,17 @@
  *   -O pileups.table
  * </pre>
  *
+ * Although the sites (-L) and variants (-V) resources will often be identical, this need not be the case.  For example,
```

Comment:
Typically these files would only be different if you were trying to use a subset of the variants?  I.e. -L was a subset of -V?  If so, can you add that to the doc?

Reply:
Done.